github/admin_users.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+)
+
+// createUserRequest is a subset of User and is used internally
+// by Create to pass only the known fields for the endpoint
+//
+type createUserRequest struct {
+	Login *string `json:"login"`
+	Email *string `json:"email"`
+}
+
+// CreateUser creates a new user in Github Enterprise.
```

Comment:
Please go through all files and correct the capitalization of the `GitHub` name.

If you find other instances of the `Github` capitalization, please feel free to correct those too. :smile: